code/modules/projectiles/projectile_parent.dm

```diff
@@ -517,6 +517,9 @@
 		src.tracked_blood = null
 		return
 
+	temperature_expose(datum/gas_mixture/air, exposed_temperature, exposed_volume)
+		return
```

Comment:
I assume this would prevent the flamer projectile from heating up its contents when setting everything in its turf on fire, yeah?

Reply:
that's the hope